pendant/style.css

```diff
@@ -112,3 +112,7 @@ body > .is-root-container > .wp-block-template-part > .wp-block-cover,
 	margin-right: auto !important;
 	width: inherit;
 }
+
+.custom-404-wp-search {
+	max-width: 500px;
+}
```

Comment:
is there a way to set max-width: 500px without a custom class?
small screens should have full width, and large screens upto 500px.

Reply:
Maybe you can put the search block inside a column, you can control the width of those

Reply:
This: `"width":500,"widthUnit":"px"` on the navigation block seems to have done the trick to keep it at 500px.